mklishevych.js

```diff
@@ -0,0 +1,13 @@
+function my(a, b, c, x) {
+    var ratio = x / (a + b + c);
```

Comment:
divide by zero
nan
negative

btw not in folder fork with your name

Reply:
Probably task definition is not strict enough, because it's not clear how the following is possible.
How can it be zero, if negative numbers are not allowed?
How NaN is possible if passed arguments are numbers?

Reply:
a, b, c could be zero
as I remember I wrote about from 0
that means negative is not allowed, from rules, it's part of question)